lib/emque/consuming/adapters/rabbit_mq/worker.rb

```diff
@@ -73,11 +77,54 @@ def process_message(delivery_info, metadata, payload)
               )
               ::Emque::Consuming::Consumer.new.consume(:process, message)
               channel.ack(delivery_info.delivery_tag)
-            rescue StandardError
-              channel.nack(delivery_info.delivery_tag)
+            rescue StandardError => ex
+              if enable_delayed_message
+                publish_to_delayed_message(delivery_info, metadata, payload)
```

Comment:
what happens if this method fails? since in the rescue handler, the message will remain in limbo possibly? Do we need to guarantee the ack happens?